source/features/repo-age.tsx

```diff
@@ -33,13 +59,7 @@ const getFirstCommit = cache.function(async (): Promise<[string, string] | undef
 		return [select('.commit-tease-sha + span relative-time')!.attributes.datetime.value, commitUrl];
 	}
 
-	const commit = await fetchDom(
-		`${getRepoURL()}/commits?after=${commitSha}+${commitsCount - 2}`,
-		'.commit'
-	);
-	const timeStamp = select('relative-time', commit)!.attributes.datetime.value;
-	const {pathname} = select<HTMLAnchorElement>('a.message', commit)!;
-	return [timeStamp, pathname];
+	return getRepoAge(commitSha, commitsCount - Math.min(6, commitsCount));
```

Comment:
What does this mean? That's 6?

Reply:
We was all the last commit - 6 (Which is 5 its complicated but I thnk 0 and 1 are the same)
I will double check give me a min

Reply:
Yes 0 and 1 are the same

Try it move the 1886 to 1887 and 1888
```
{
  repository(owner: "sindresorhus", name: "refined-github") {
    defaultBranchRef {
      target {
        ... on Commit {
          history(first: 5, after: "208835085ee2a78b6a9d4f30b1407c7eac1b88b3 1886") {
            nodes {
              authoredDate
              committedDate
              resourcePath
            }
          }
        }
      }
    }
  }
}
```

Reply:
Why 6?

Reply:
We want 5, 1 =0 (I think)

Reply:
OK the math is like this. We are at commit 10 and we want AFTER commit 10 so we need to start from number 9 = (10 - (1 + 5) )

Reply:
So the math belongs to `getRepoAge`, not outside.
5 is inside and 6 is outside; doesn't make sense.

```suggestion
	return getRepoAge(commitSha, commitsCount);
```

Reply:
We have commit number 1 we want the last 5. There are 1000 commits. If we do 1 + 1000 (The commit count) we get commit number 1001 which does not exist.
So we need to go back 1 and then another 5 to get the last 5.

Reply:
The latest version looks right